source/content/wp-config-php.md

```diff
@@ -5,43 +5,64 @@ cms: "WordPress"
+Never put the database connection information for a Pantheon database within your `wp-config.php`. These credentials will change.
+
+If you experience connection errors, ensure that you are running the latest version of WordPress core and have the correct `wp-config.php` file for Pantheon.
 
 </Alert>
 
-## Local Database Configuration for Development
-If you are also developing locally and need to configure WordPress for your desktop environment, we recommend you create a wp-config-local.php file. This will be ignored by Pantheon and should not be tracked by version control by default since it's in the .gitignore file.
+### Pantheon Platform Settings in wp-config-pantheon.php
+
+In order to get the latest WordPress upstream updates while at the same time avoiding merge conflicts, Pantheon includes `wp-config-pantheon.php`.
+
+Do not edit `wp-config-pantheon.php`. It includes database and environment configuration that the platform uses and that Pantheon maintains.
+
+## wp-config-local for Local Database Development Configuration
+
+The Pantheon WordPress upstream includes a sample configuration file for [local development](/local-development).
+
+If you are developing locally and need to configure WordPress for your desktop environment, make a copy of `wp-config-local-sample.php` called `wp-config-local.php`. This file is listed in the `.gitignore` file and will not be tracked by version control by default.
+
+To make local development easier, when WordPress detects `wp-config-local.php`, WordPress uses the configuration in that file instead of the settings in `wp-config.php`.
 
 ## Pantheon's WordPress Config
 
 <Accordion title="View Pantheon's WordPress Configuration" id="pantheon-wp-config-php" icon="wrench">
 
-GITHUB-EMBED https://github.com/pantheon-systems/wordpress/blob/master/wp-config.php php GITHUB-EMBED
+You can also find this file on [GitHub](https://github.com/pantheon-systems/WordPress/blob/default/wp-config.php).
+
+GITHUB-EMBED https://github.com/pantheon-systems/WordPress/blob/default/wp-config.php php GITHUB-EMBED
 
 </Accordion>
 
-<Alert tile="Note" type="info">
+<Alert title="Note" type="info">
 
 `$_SERVER['SERVER_NAME']` should *not* be used to set `WP_HOME` or `WP_SITEURL`. For more information, see [SERVER_NAME and SERVER_PORT on Pantheon](/server_name-and-server_port).
 
 </Alert>
 
+## Environment-specific Configuration
 
+### Use the WP_ENVIRONMENT_TYPE Function to Perform Actions Based on Environment
 
-## Frequently Asked Questions
+<Partial file="wp_get_environment_type.md" />
 
-### How can I write logic based on the Pantheon server environment?
-Depending on your use case, there are two possibilities:
+### Write Logic Based on the Pantheon Server Environment
```

Comment:
this might need to be updated

Reply:
@greg-1-anderson I included [L31](https://github.com/pantheon-systems/documentation/pull/5919/files#diff-2ac3a791b02a0cc812f3ece836417f2eR31) and tried to rework this section as a start. It could probably use a more thorough review, but I'm leaving it like this for now in an effort to get some documentation live. Should this (or any other section (I tried to highlight them in comments)) be more completely updated before this PR goes live or can that go into its own issue/pr?

Reply:
I think this is fine; just make the one more suggestion I missed last time and I think it's good to go.